pynestml/visitors/ast_function_call_visitor.py

```diff
@@ -66,10 +81,11 @@ def visit_simple_expression(self, node):
 
             if isinstance(return_type, TemplateTypeSymbol):
                 # error: return type template not found among parameter type templates
-                assert(False)
+                assert (False)
```

Comment:
```suggestion
                assert False, "return type template not found among parameter type templates"
```
Should this raise an exception or be an assertion? (Programmer error or user error?)

Reply:
This change is a result of the auto-format of the code by the editor.  The same if condition is also present on line 76. Isn't this redundant?

Reply:
The variable ``return_type`` can change inbetween.